### PR TITLE
notepadqq: bump version

### PR DESCRIPTION
**Pull request by FuRuYa7**
notepadqq-2.0.0-beta

### CODE REVIEW

File: app-editors/notepadqq/notepadqq-2.0.0.recipe [hunk 1]
```diff
@@ -6,8 +6,9 @@ HOMEPAGE="http://notepadqq.altervista.org/"
 COPYRIGHT="2012-2019 Notepadqq Project"
 LICENSE="GNU LGPL v3"
 REVISION="3"
```
Comment (diversys) [inline app-editors/notepadqq/notepadqq-2.0.0.recipe:8]: Please reset revision number to 1.

**Comment by diversys** [comment]
I think the recipe should be renamed to `notepadqq-2.0.0~beta.recipe`

**Comment by FuRuYa7** [comment]
REVISION="1"

**Comment by FuRuYa7** [comment]
The file name was not recognized by "notepadqq-2.0.0-beta.recipe".
I haven't tried "notepadqq-2.0.0~beta.recipe", so I'll try it.

Thanks.

**Comment by FuRuYa7** [comment]
Thank you for the good information.

**Comment by Begasus** [comment]
> --> The conclusion didn't go well.

You could look at how other packages handle this :) 
Example: https://github.com/haikuports/haikuports/blob/4f180bdb94db9749ae800820b271e262a12754aa/app-office/whid/whid-2.0.2%7Ebeta.recipe#L15

https://github.com/haikuports/haikuports/search?q=portversion+%7Ebeta

**Comment by JuBan1** [comment]
Hey everyone. Thanks for showing interest in my Nqq OTE fork. I poured a lot of work into it in 2019 :)

But if you package it you have to be aware that your users won't get any bug fixes or new features (unless they implement it themselves). I don't support it at all, and I'm not using it myself anymore since ~2020. The same goes for the original Nqq by the way; it's effectively dead.

**Comment by waddlesplash** [comment]
>  I haven't tried "notepadqq-2.0.0~beta.recipe", so I'll try it.

Just use this recipe name, and then skip $portVersion in the recipe if you have to and just hard-code the version and directory names instead. It's more important that the recipe has the correct name.

**Comment by waddlesplash** [comment]
> I don't support it at all, and I'm not using it myself anymore since ~2020. The same goes for the original Nqq by the way; it's effectively dead.

Yeah, we did notice; but having a Notepad++-like editor is very nice, even if it isn't maintained anymore. Maybe someone will take interest in it if we package it anyway...

**Comment by FuRuYa7** [comment]
I build and used "Notepadqq-OTE" (JuBan1/notepadqq). wonderful. We will continue to use useful tools in the future.

Thank you for your hard work.

**Comment by FuRuYa7** [comment]
Corrected the file name and the settings that accompany it. 

```
notepadqq-2.0.0~beta.recipe
patches/notepadqq-2.0.0-beta.patchset
```

Thank you for your advice.

**Comment by FuRuYa7** [comment]
#### "Notepadqq-OTE" (JuBan1/notepadqq)

```
notepadqq-1.4.8~ote.recipe
patches/notepadqq-1.4.8-ote.patchset
```

"notepadqq-2.0.0~beta" is slow to start the application. Even if I upgraded the version, the usability became worse. I changed to "notepadqq-OTE".

**Comment by korli** [comment]
the "~" suffix is only for pre-release versions. Just use one dot instead.

**Comment by FuRuYa7** [comment]
Changed "~" in the file name to "." 

```
notepadqq-1.4.8.ote.recipe
patches/notepadqq-1.4.8.ote.patchset
```

**Pull request merged**